psi4/src/psi4/libmints/basisset.cc

```diff
@@ -571,133 +574,122 @@ std::shared_ptr <BasisSet> BasisSet::zero_ao_basis_set()
     return new_basis;
 }
 
-std::shared_ptr<BasisSet>
-BasisSet::construct_ecp_from_pydict(std::shared_ptr <Molecule> mol, py::dict pybs, const int forced_puream){
+std::shared_ptr<BasisSet> BasisSet::construct_from_pydict(const std::shared_ptr <Molecule> &mol, py::dict pybs, const int forced_puream){
 
     std::string key = pybs["key"].cast<std::string>();
     std::string name = pybs["name"].cast<std::string>();
     std::string label = pybs["blend"].cast<std::string>();
-    std::string message = pybs["message"].cast<std::string>();
```

Comment:
This is covered by the main basis set message, or do I need to write a modified ecp-extracted-from message string? I may well have left that code overly concise.